frontend/src/pages/profile/libs/helpers/get-progress-style-class.helper.ts

```diff
@@ -0,0 +1,19 @@
+import { type ValueOf } from 'shared/build/index.js';
+
+import { ProgressStatusClass } from '../enums/enums.js';
+
+const getProgressStyleClass = (
+  progress: number,
+): ValueOf<typeof ProgressStatusClass> => {
+  if (progress <= 0) {
```

Comment:
I guess you can use these constants `frontend/src/pages/profile/libs/constants/constants.ts` here

Reply:
[245d853](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/156/commits/245d853e0997b9527aa4015c1f21a11f3eb19d9c)